library/build.gradle

```diff
@@ -29,6 +29,7 @@ android {
 dependencies {
     compile project("${dbflow_project_prefix}flowcore")
     compile "com.android.support:support-annotations:21.0.0"
+    compile 'com.android.support:support-v4:+'
```

Comment:
I would advise against using + in dependencies. Also this adds a support library dependency. Are loaders only available on 3.0+?